internal/data/circle_transfer_requests.go

```diff
@@ -141,15 +141,15 @@ func (m CircleTransferRequestModel) GetPendingReconciliation(ctx context.Context
 	return m.GetAll(ctx, sqlExec, queryParams)
 }
 
-const baseQuery = `
+const baseCircleQuery = `
```

Comment:
Cleanup: renamed this because it can be accessed from other parts of the `data` package.